### PR TITLE
Add a new label for when the menu has been ordered

### PR DESCRIPTION
**Pull request by Gkolocsar**
The label is now updated to indicate when  the following day menu has been ordered too. 

I'll try to update the day labels when selecting the menus and the special days ones, but it looks that bigger changes need to be done in order to do that.

Can you still please review this far? @andresmoschini @AlphaGit @sofipacifico

### CODE REVIEW

File: source/CommonJobs/CommonJobs.Application.MyMenu/GetEmployeeMenuCommand.cs [hunk 1]
```diff
@@ -47,13 +47,22 @@ public override EmployeeMenuDTO ExecuteWithResult()
 
             var menuDefinition = ExecuteCommand(new GetMenuDefinitionCommand(employeeMenu.MenuId));
 
-            var lastOrder = RavenSession.Load<MenuOrder>(MenuOrder.GenerateId(employeeMenu.MenuId, menuDefinition.LastOrderDate));
-            if (lastOrder != null)
-            {
-                lastOrder.IsOrdered = true;
-            }
+            var today = DateTime.Now.Date;
```
Comment (andresmoschini) [inline source/CommonJobs/CommonJobs.Application.MyMenu/GetEmployeeMenuCommand.cs]: I think that in general we was trying to make commands non-dependent on `DateTime.Now`, what do you think about to send current date as a parameter?

File: source/CommonJobs/CommonJobs.Application.MyMenu/GetEmployeeMenuCommand.cs [hunk 2]
```diff
@@ -47,13 +47,22 @@ public override EmployeeMenuDTO ExecuteWithResult()
...
+            var tomorrow = today.AddDays(1);
+            var todayOrderId = MenuOrder.GenerateId(employeeMenu.MenuId, today);
+
+            var todayOrder = RavenSession.Load<MenuOrder>(todayOrderId);
+
+            // IS THIS NECESSARY? The orders are created with IsOrdered = true
+            //if (todayOrder != null)
+            //{
+            //    todayOrder.IsOrdered = true;
+            //}
```
Comment (andresmoschini) [inline source/CommonJobs/CommonJobs.Application.MyMenu/GetEmployeeMenuCommand.cs]: It seems to be a message for me, maybe this code was there in order to support old documents without `IsOrdered` property. If current data and code confirms that it is not necessary, please remove it.

File: source/CommonJobs/CommonJobs.Application.MyMenu/GetEmployeeMenuCommand.cs [hunk 3]
```diff
@@ -47,13 +47,22 @@ public override EmployeeMenuDTO ExecuteWithResult()
...
+
+            // If the last order generated is from tomorrow, then we should tell the user
+            var hasTomorrowBeenOrdered = (menuDefinition.LastGeneratedOrderDate.Date == tomorrow);
```
Comment (andresmoschini) [inline source/CommonJobs/CommonJobs.Application.MyMenu/GetEmployeeMenuCommand.cs]: I am not sure, but what do you think about it:

```csharp
var hasTomorrowBeenOrdered = (menuDefinition.LastGeneratedOrderDate.Date >= tomorrow);
```

File: source/CommonJobs/CommonJobs.Application.MyMenu/GetEmployeeMenuCommand.cs (hunk 1 above)
Comment (Gkolocsar) [inline source/CommonJobs/CommonJobs.Application.MyMenu/GetEmployeeMenuCommand.cs]: Sounds good

File: source/CommonJobs/CommonJobs.Application.MyMenu/GetEmployeeMenuCommand.cs (hunk 2 above)
Comment (Gkolocsar) [inline source/CommonJobs/CommonJobs.Application.MyMenu/GetEmployeeMenuCommand.cs]: :+1:

File: source/CommonJobs/CommonJobs.Application.MyMenu/GetEmployeeMenuCommand.cs (hunk 3 above)
Comment (Gkolocsar) [inline source/CommonJobs/CommonJobs.Application.MyMenu/GetEmployeeMenuCommand.cs]: Better!

**Comment by andresmoschini** [comment]
Great @Gkolocsar! I did not review it in deep, but looks fine.

Please, consider my comments and another one (maybe little annoying), since the changes in `my-menu-order-page.js` and `my-menu.js` are really simple ones, could you also update (manually) related `.ts` files? (It will be nice to, in the near future, update to the latest TypeScript version to take advantage of this language)

**Comment by Gkolocsar** [comment]
Sure, will update the TS files.

**Pull request merged**